tutorials/plot_metadata_epochs.py

```diff
@@ -0,0 +1,118 @@
+"""
```

Comment:
many lines are too long in this tutorial no?

Reply:
Please add these to `documentation.rst` so we can see where they will live